smart-contracts/package.json

```diff
@@ -34,12 +34,12 @@
   },
   "scripts": {
     "remix": "remixd -s contracts --remix-ide http://localhost:8080 --read-only",
-    "test": "cross-env NODE_ENV=test truffle test --network development",
+    "test": "cross-env NODE_ENV=test truffle migrate && truffle test",
```

Comment:
For some reason all tests fail if I don't migrate first.  ...why?

Reply:
Are you referring to the tests in `smart-contracts` or `unlock-app`? I know with the `unlock-app` tests there were a bunch that would fail unless `npm run dev` was run first...I believe that was to do with contracts needing compilation. But maybe that's outdated now and no longer the case.

Reply:
unlock-app, on the second+ run only. (it works only with the first run)

Reply:
This is weird. I need to think about what's happening here.